src/decorators.ts

```diff
@@ -22,7 +22,7 @@ export function Transform(transformFn: (value: any) => any, options?: TransformO
  */
 export function Type(typeFunction?: (type?: TypeOptions) => Function) {
     return function(target: any, key: string) {
-        const type = (Reflect as any).getMetadata("design:type", target, key);
+        const type = Reflect.getMetadata("design:type", target, key);
```

Comment:
this will cause problems on users because they don't have `Reflect` with such methods until they install specific packages. But I don't them to have compiler errors from this package right after they install it.